src/common/types/value.cpp

```diff
@@ -26,6 +26,7 @@ Value::Value(string_t val) : Value(string(val.GetData(), val.GetSize())) {
 }
 
 Value::Value(string val) : type(TypeId::VARCHAR), is_null(false) {
+	str_value = val;
```

Comment:
This is not necessary, no? It is already set below.

Reply:
Yeah, it's a old code that I forgot to remove it (good catch).